### ISSUE TITLE
Not working with grafana 9.5.2

### ISSUE THREAD

**Issue by Bixilon** [Checkmk/grafana-checkmk-datasource#194]
I run the current version of grafana (version 9.5.2) on arm64. I installed the latest version (3.0.1-alpha1; also tried 3.0.0) by unzipping the release files and copying them to `/var/lib/grafana/plugins/`.

It works, the plugin gets loaded, but grafana crashes pretty fast when adding a new datasource (An unexpected error happened):
![Screenshot_20230515_211128](https://github.com/Checkmk/grafana-checkmk-datasource/assets/26258520/7f4c51c4-2b7d-4a9b-8632-3f3e97f3b309)

```
act-dom.production.min.js:216 Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Wl (react-dom.production.min.js:237:48)
    at ec (react-dom.production.min.js:170:408)
    at index-a86253bb.esm.js:677:7
    at je (floating-ui.dom.browser.min.mjs:1:6624)
    at index-a86253bb.esm.js:692:28
    at index-a86253bb.esm.js:702:5
    at Rd (react-dom.production.min.js:259:307)
    at g.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at nu (react-dom.production.min.js:122:325)
    at lc (react-dom.production.min.js:252:279)
P
```

It worked after I tried it several times. Now I added the data source, but it keeps crashing:

![Screenshot_20230515_211312](https://github.com/Checkmk/grafana-checkmk-datasource/assets/26258520/4348c534-95a7-4e38-8e40-72666a204c7e)

Did I something wrong?

Hope for your help

Thanks!

**Comment by TheSpacyCat** [comment]
Hey!

Doesn't seem to me like you did something wrong, but sadly, neither can I reproduce your issue on my development machine.

Could it be that you are upgrading from an older release? If so, could you show me a how the older version was configured?

**Comment by Bixilon** [comment]
No, grafana is a fresh instance, running on a raspberry pi.

**Comment by BenediktSeidl** [comment]
I also looked into that, and can not reproduce this.

Can you describe what you do to see the first error message? (There are multiple ways to add a new data source)
Which browser and operating system do you use?
Have you tried to add another data source plugin? Are they able to show the first dialog?
Can you try to access the page from another computer/browser?
Can you try to install grafana on another computer?

**Comment by Bixilon** [comment]
I am going to `Connections` -> search of `checkmk` -> blue button `Create new datasource`

Then the form does not load, after reloading the page the edit menu shows properly. 

It crashes as soon as I click on any dropdow menu.


Interestingly, there is an 404 request to `10.10.7.1:3000/api/gnet/plugins/tribe-29-checkmk-datasource`


Even more interstingly: It also crashes when adding an InfluxDB data source, thats "a good thing". Did not try that on this grafana instance before. I guess  I will reach out to grafana devs…

**Comment by godspeed-you** [comment]
Hi @Bixilon,
did you got any response from the Grafana devs?

**Comment by Bixilon** [comment]
They actually closed the issue ("its checkmks problem"), which is simply stupid. I opened another one, but no response yet: https://github.com/grafana/grafana/issues/69324

**Comment by godspeed-you** [comment]
Maybe it's helpful if you're also mentioning influxdb. Did you encounter the problem on any other integration (except Checkmk and influxdb)?

**Comment by Bixilon** [comment]
Its not about influx db or any data source. It is a generic bug - grafana crashes when clicking on a dropdown
